Application/mobile/app_code/lib/pages/article/list_state.dart

```diff
@@ -48,9 +108,6 @@ class ArticleListState extends State<ArticleListPage> {
       return [];
     } catch (err, stacktrace) {
       if (context.mounted) {
-        setState(() {
```

Comment:
Why removing this ?

Reply:
its never set to true, so we don't need to set to false.

Reply:
You need to keep it and put it to true before doing a request